shell.nix

```diff
@@ -45,10 +57,36 @@ let inherit (nixroot) stdenv pkgs lib
     '';
   };
 
+  libkeyfinder = stdenv.mkDerivation {
```

Comment:
Does `libkeyfinder` from nixpkgs not work?

Reply:
I'm using 19.03 and it does not have it. 20.03 is broken here, but I will use upstream if there is a package for it.